docs/reference/ocm_download_cli.md

```diff
@@ -91,6 +94,15 @@ This is handled by download handler. Their usage can be enabled with the <code>-
 option. Otherwise the resource as returned by the access method is stored.
 
 
+If the verification store is enabled, resources downloaded from
```

Comment:
How is this functionality related in any way to the word "store"? I would assume something like `--verify` or something would be more akin to this functionality. Store, means storing something somewhere. This implies that it's just verification of some sort. And if it IS stored, then it should be explained where, for how long, why, and it what format and in under what circumstances the store is invalidated.

Reply:
Indeed, the store stores the verified component versions together with the verified digests, therefore store seems to be applicable.  If a CV is verified and not changed then it is verified forever. To invalidate your current verifications you just need to remove the store file or just use another one.

Reply:
Ah, I see. I might have misunderstood the functionality of the store then. Thanks.